vdev/src/commands/package/msi.rs

```diff
@@ -0,0 +1,58 @@
+use anyhow::Result;
+
+#[cfg(windows)]
+use {
+    crate::{app, util},
+    std::env,
+    std::fs,
+    std::iter::once,
+    std::path::Path,
+};
+
+/// Create a .msi package for Windows
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        #[cfg(windows)]
```

Comment:
There should be a `#[cfg(not(windows))]` block as well that prints an error that nothing was done. Ideally this subcommand would not even be present on non-Windows, but the macro makes that difficult.

Reply:
Will add a message implementing this feedback